src/driver/tests/experiment_driver_test.py

```diff
@@ -108,6 +114,28 @@ def test_experiment_driver_with_m3_first_round_experimental_design(self):
             result = experiment_driver.execute()
             self.assertEqual(result.shape[0], 5184)
 
+    @patch(
+        "wfa_planning_evaluation_framework.driver.experiment.ExperimentalTrial",
+        new=FakeExperimentalTrial,
+    )
+    def test_experiment_driver_with_analysis_example_experimental_design(self):
+        with TemporaryDirectory() as d:
+            data_design_dir = d + "/data"
+            output_file = d + "/output"
+            intermediate_dir = d + "/intermediates"
+            data_design_generator = SyntheticDataDesignGenerator(
+                data_design_dir, 1, analysis_example_data_design.__file__, False
+            )
+            data_design_generator()
+            rng = np.random.default_rng(seed=1)
+            experimental_design = analysis_example_experimental_design.__file__
+            experiment_driver = ExperimentDriver(
+                data_design_dir, experimental_design, output_file, intermediate_dir, rng
+            )
+            result = experiment_driver.execute()
+            print(result.shape[0])
```

Comment:
Remove print statement and uncomment assertion.

Reply:
Done.